client/src/main/groovy/de/gesellix/docker/client/image/ManageImageClient.groovy

```diff
@@ -76,16 +76,21 @@ class ManageImageClient implements ManageImage {
     }
 
     @Override
-    build(InputStream buildContext, query = ["rm": true], DockerAsyncCallback callback = null) {
+    build(InputStream buildContext, query = ["rm": true], buildOptions = [:], DockerAsyncCallback callback = null) {
         log.info "docker build"
         def async = callback ? true : false
         def actualQuery = query ?: [:]
         queryUtil.jsonEncodeBuildargs(actualQuery)
-        def response = client.post([path              : "/build",
-                                    query             : actualQuery,
-                                    body              : buildContext,
-                                    requestContentType: "application/octet-stream",
-                                    async             : async])
+        buildOptions = buildOptions ?: [:]
+        def request = [path              : "/build",
+                       query             : actualQuery,
+                       body              : buildContext,
+                       requestContentType: "application/octet-stream",
+                       async             : async]
+        if (buildOptions.EncodedRegistryConfig) {
```

Comment:
I'm aware of the different header names for the same use case, but I would prefer to have a consistent Java api (I'm slowly working towards a more type safe approach). So, please rename the build option `EncodedRegistryConfig` to `EncodedRegistryAuth`, please.

Reply:
Ok, no problem. I will check for the integration test tonight or tomorrow. Thanks for your review